lib/netsuite/records/serialized_inventory_item_locations.rb

```diff
@@ -0,0 +1,41 @@
+module NetSuite
+  module Records
+    class SerializedInventoryItemLocations
```

Comment:
I created this based on SerializedInventoryItemLocationsList: http://www.netsuite.com/help/helpcenter/en_US/srbrowser/Browser2015_1/schema/other/serializedinventoryitemlocations.html?mode=package.

Could it be "actions"?